streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicManager.java

```diff
@@ -578,8 +579,29 @@ private void readyTopics(final Set<NewTopic> topicsToCreate,
                     }
                 }
             }
+
+            if (!topicsNotReady.isEmpty()) {
```

Comment:
Should this go after the for-loop? I don't think we need to retry-backoff after verifying a single topic, but should be sufficient to check once for the whole "batch" of topics?